### ISSUE TITLE
Jinx mismatch between google sheet and BOTC wiki

### ISSUE THREAD

**Issue by fuine** [Skateside/pocket-grimoire#141]
I'm working on Polish translation and I've discovered a mismatch between jinxes in the spreadsheet and BOTC wiki. I haven't checked all jinxes, so I'm not sure how big is the mismatch between these two sources or which of them is the newest/closest to the official game. As an example take jinxes with [Leviathan](https://wiki.bloodontheclocktower.com/Leviathan) as the target and Soldier as the trick:

Wiki:
> If the Leviathan is in play, the Soldier is safe from all evil abilities.

Spreadsheet:
> If Leviathan nominates and executes the Soldier, the Soldier does not die.

I know that Leviathan is an experimental character and I'm not aware of any mismatches between non-experimental characters but is there a way to check how big the mismatch is and correct it?

**Comment by Skateside** [comment]
Yeah, the jinxes document is pretty out-of-date at the moment. I'm slowly working through it but it's just going to take a while.

Thanks for highlighting it though

**Comment by fuine** [comment]
Yea, no worries, I have a python script which can generate the english csv file based on the official `data.json` file and diff the old one with the new one to highlight the changes - I can post it here if that would help you in any way

**Comment by Skateside** [comment]
I've already got the differences, it's just a case of going through them all. 120ish jinxes across 25 languages just takes a while

Thanks though

**Comment by Skateside** [comment]
I've replaced the old version of the jinxes file with a new one that _does_ match the wiki
https://docs.google.com/spreadsheets/d/193DMlJzVSzArj1hV1DF6jcr-NsGRaecAy1ahLflu-Qo/edit?usp=sharing
